src/Symfony/Component/Yaml/Inline.php

```diff
@@ -127,6 +127,8 @@ public static function dump($value, int $flags = 0): string
                 return self::dumpNull($flags);
             case $value instanceof \DateTimeInterface:
                 return $value->format('c');
+            case $value instanceof \UnitEnum:
+                return sprintf("!php/const '%s::%s'", \get_class($value), $value->name);
```

Comment:
The simple quotes are not required, isn't it? Let's drop them if you confirm.

Reply:
Yep you're right, removed!